process-engine/bpmn-studio
# ✨Add Save As Menu Entry

## Changes

1.  Add Save As Menu Entry for Electron
2. Add Hotkey Handler for Save As
3. Add the Possibility to Save the Diagram to a new Location

## Issues

Closes #371 

PR: #1496 

## How to test the changes

- open a diagram
- make some changes
- click on "Save as" on the context menu or use CTRL/CMD+SHIFT+S hotkey
- choose any location and diagram name
- see that the diagram has been saved and is opened on the solution explorer as a single diagram
